frontend/src/action-creator.sync.js

```diff
@@ -120,15 +120,16 @@ export const prependBreadcrumbs = prependBreadcrumbs => ({ type: `${PREPEND}/${B
 export const appendBreadcrumbs = appendBreadcrumbs => ({ type: `${APPEND}/${BREADCRUMBS}`, appendBreadcrumbs })
 export const resetBreadcrumbsAppFeature = () => ({ type: `${RESET}/${BREADCRUMBS}/${APP_FEATURE}` })
 
-export const RESEARCH = 'Research'
-export const setResearch = newResearch => ({ type: `${SET}/${RESEARCH}`, newResearch })
-export const resetResearch = () => ({ type: `${RESET}/${RESEARCH}` })
-export const appendResearch = appendResearch => ({ type: `${APPEND}/${RESEARCH}`, appendResearch })
-export const RESEARCH_ELEMENTS = 'ResearchElements'
-export const setNbElementsResearch = nbElements => ({ type: `${SET}/${RESEARCH_ELEMENTS}`, nbElements })
-export const KEY_WORD_RESEARCH = 'KeyWordResearch'
-export const setKeyWordResearch = keyWordResearch => ({ type: `${SET}/${KEY_WORD_RESEARCH}`, keyWordResearch })
-export const RESEARCH_ELEMENTS_BY_PAGE = 'ResearchElementsByPage'
-export const setNbElementsByPage = nbElementsByPage => ({ type: `${SET}/${RESEARCH_ELEMENTS_BY_PAGE}`, nbElementsByPage })
-export const RESEARCH_PAGE = 'ResearchPage'
-export const setNbPage = nbPage => ({ type: `${SET}/${RESEARCH_PAGE}`, nbPage })
+export const SEARCH = 'Search'
+export const resetSearch = () => ({ type: `${RESET}/${SEARCH}` })
```

Comment:
You had asked to do this function because it could be useful, but we still don't use it, so I left the reset for the whole object, but I can change it if you think it's better.